pkg/test/common/utils.go

```diff
@@ -0,0 +1,25 @@
+package common
+
+import (
+	"net/url"
+
+	"gopkg.in/resty.v1"
+)
+
+func Location(baseURL string, resp *resty.Response) string {
```

Comment:
Yes, we need to start pulling in only needed things. Although this goes against golang development model but then upstream folks don't necessarily follow pkg export discipline.